src/hooks/useLoadVocabularyItems.ts

```diff
@@ -0,0 +1,59 @@
+import { ARTICLES, DOCUMENT_TYPES } from '../constants/data'
+import { VocabularyItem, ENDPOINTS } from '../constants/endpoints'
+import { getFromEndpoint } from '../services/axios'
+import useLoadAsync, { Return } from './useLoadAsync'
+
+export interface AlternativeWordFromServer {
+  article: number
+  alt_word: string
+}
+
+export interface VocabularyItemFromServer {
+  id: number
+  word: string
+  article: number
+  document_image: Array<{ id: number; image: string }>
+  audio: string
+  alternatives: AlternativeWordFromServer[]
+}
+
+export const formatVocabularyItemFromServer = (
+  vocabularyItemFromServer: VocabularyItemFromServer,
+  apiKey?: string
+): VocabularyItem => ({
+  ...vocabularyItemFromServer,
+  documentType: apiKey ? DOCUMENT_TYPES.lunesProtected : DOCUMENT_TYPES.lunesStandard,
```

Comment:
Should be `vocabularyItemType` or imo even just `type`.

Reply:
oh yea true it gets added in the front end